### DISCUSSION TITLE
Trigger feed header colors

### DISCUSSION THREAD

**Discussion by feedspeed** [levito/tt-rss-feedly-theme#113] [Q&A]
With this theme all feeds have colored headers. I've noticed that is seems that the color is periodically derived from the main color of the icon image, is that true? 
If true; can this 'header color generation' be manually triggered somehow?

**Comment by levito** [comment]
Hi feedspeed, this happens inside of tt-rss and it's not part of the theme. No idea if this can be triggered in some way. You can right-click on a feed and choose "Edit feed", then go to the "Icon" tab and remove it. Maybe this triggers refetching the icon and regenerating the color. But I haven't tried it out.

**Comment by feedspeed** [comment]
Thanks! Whenever you've changed something, like the icon or even the feed title, it changes to a random colour. Then, sometimes, not always it changes after a longer period.
And I know your theme is just CSS, but hoped somebody knew were to look in the TT-RSS code. As this is the only theme who seems to use coloured headers.

Anyway; thank you for your great theme!
